program_info/win_install.nsi.in

```diff
@@ -171,6 +307,27 @@ Section /o "Desktop Shortcut" DESKTOP_SHORTCUTS
 
 SectionEnd
 
+
+!define APP_ID "@Launcher_CommonName@.App"
+!define APP_EXE "@Launcher_APP_BINARY_NAME@.exe"
+!define APP_ICON "$INSTDIR\${APP_EXE},0"
+!define APP_DESCRIPTION "@Launcher_DisplayName@"
+!define APP_NAME "@Launcher_DisplayName@"
+!define APP_CMD_TEXT "Prism Launcher instance"
```

Comment:
technically, they're modpack formats

Reply:
they are, but since this lets you open them and import them "as" an instance I'm not sure what else to name the command.

Reply:
Minecraft modpack maybe

Reply:
changed.